### DISCUSSION TITLE
DreamBerd vHas vBeen vRenamed cTo nDreamBerd

### DISCUSSION THREAD

**Discussion by TodePond** [TodePond/GulfOfMexico#483] [Announcements]
vWelcome cTo adjAnother adjWeekly nDreamBerd nRelease nNotes.

## adjNew nName

aFirst, pWe'vRe qReally adjPleased cTo vAnnounce nDreamberd's adjNew nName: nDreamBerd.

pWe vPromise pWe vWon't vChange pIt advAgain.

## adjSemantic nNaming

pWe'vRe qAlso adjExcited cTo vIntroduce aA qBrand adjNew nFeature. pWe vCall pIt adjSemantic vNaming, cOr adjHungarian nNotation.

advNow pIt's adjEasier cTo vRead nDreamberd cThan advEver nBefore.

```java
const const sName = "Lu"!
const const iAge = 29!
const const bHappy = true!
```

pWe qEven vSupport g_nGlobals.

```java
const const g_fScore = 4.5!
```

pWe vLook advForward cTo vSeeing vYou cFor adjNext nWeek's nRelease nNotes.

<hr /><em>This discussion was created from the release <a href='https://github.com/TodePond/DreamBerd/releases/tag/'>DreamBerd vHas vBeen vRenamed cTo nDreamBerd</a>.</em>

**Comment by TheKodeToad** [comment]
pWhat

**Comment by mybearworld** [comment]
iHuh. vAm pI vHaving artA nDéjà vu?

**Comment by TodePond** [comment]
eOops

**Comment by ghost** [comment]
pI eWant dTo adjAsk eYou cA vFew dQuestions:
* pWhy eDid iYou kChange pIt?

**Reply by mybearworld** [reply]
adjSemantic vNaming vIs adjGreat, dThat's iWhy
